docs/basic_training/intro.es.md

```diff
@@ -0,0 +1,274 @@
+
+Si no se especifica lo contrario, los procesos se ejecutan en la computadora local. El ejecutor local es muy útil para desarrollar flujos y con fines de prueba; sin embargo, para los flujos computacionales del mundo real, a menudo se requiere una plataforma de alto rendimiento (HPC) o la nube.
+
+En otras palabras, Nextflow proporciona una abstracción entre la lógica funcional del flujo y el sistema de ejecución (o runtime) subyacente. Por lo tanto, es posible escribir un flujo que se ejecute sin problemas en su computadora, un clúster o la nube, sin ser modificada. Simplemente defina la plataforma de ejecución de destino en el archivo de configuración.
+
+<figure markdown>
+
+![Execution abstraction](img/execution_abstraction.png)
+
+</figure>
+
+### Lenguaje de escritura
+
+Nextflow implementa un DSL declarativo que simplifica la escritura de flujos de trabajo de análisis de datos complejos como una extensión de un lenguaje de programación de propósito general.
+
+Este enfoque hace que Nextflow sea flexible: brinda los beneficios de un DSL conciso para el manejo de casos de uso frecuente con facilidad **y** da la flexibilidad y el poder de un lenguaje de programación de propósito general para manejar casos más complejos en el mismo entorno. Esto sería difícil de implementar utilizando un enfoque puramente declarativo.
+
+En términos prácticos, las secuencias de comandos de Nextflow son una extensión del [lenguaje de programación Groovy] (https://groovy-lang.org/) que, a su vez, es un superconjunto del lenguaje de programación Java. Groovy puede considerarse como "Python para Java", ya que simplifica la escritura de código y es más accesible.
+
+## Tu primer script
+
+Aquí ejecutará su primer script de Nextflow (`hello.nf`), que revisaremos línea por línea.
+
+En este ejemplo sencillo, el script toma una cadena de entrada (un parámetro llamado `params.saludo`) y la divide en partes de seis caracteres en el primer proceso. El segundo proceso convierte los caracteres a mayúsculas. El resultado finalmente se muestra en la pantalla.
+
+### Nextflow code
+
+<!-- NOTE: (Phil, Jan 2023)
+We can dynamically include external files using mkdocs, as follows:
+
+```groovy title="nf-training/hello.nf" linenums="1"
+--8<-- "nf-training/hello.nf"
+```
+
+This inserts a code snippet identical to the one below, and we don't have to worry about keeping the two in sync.
+
+HOWEVER - currently the line annotations cannot be added for external files. So for now, we still need to copy the scripts.
+
+TODO: Maybe either:
+    - Rewrite docs to not use loads of annotations
+    - Wait for future versions to allow annotations with external files
+-->
+
+!!! info
+
+    Click the :material-plus-circle: icons in the code for explanations.
+
+```groovy title="nf-training/hello.nf" linenums="1"
+#!/usr/bin/env nextflow
+// (1)!
+
+params.saludo = '¡Hola mundo!' // (2)!
+saludo_ch = Channel.of(params.saludo) // (3)!
+
+process SEPARALETRAS { // (4)!
```

Comment:
In Portuguese translation they kept the process definitions in English, but I like the approach that this is translated as well. I don't know if there will be any issues with this in other parts of the tutorial. @mribeirodantas should we translate the process definition as well?

Reply:
There is code in the tutorial that refers to script files in the repository, such as most of the RNAseq section. Such code should not be translated, as this will create inconsistencies. Codes of examples that do not refer to script files in the repo should be translated, at least that's how we've approached this in Portuguese.

Maybe in the future, we will create copies of the script files for different languages.. What do you think?